src/store/actions.ts

```diff
@@ -125,5 +124,19 @@ export default {
   },
   setCoaEffectiveDate ({ commit }, coaEffectiveDate: Date) {
     commit('coaEffectiveDate', coaEffectiveDate)
+  },
+  /** Fetches stateFiling from the API and, if successful, triggers mutation. */
+  async retrieveStateFiling ({ commit }, stateFilingUrl: string): Promise<void> {
+    const filing = stateFilingUrl && await LegalServices.fetchFiling(stateFilingUrl)
+    const filingType = filing?.header?.name as FilingTypes
+    return new Promise((resolve, reject) => {
+      if (!filing || !filingType) {
+        console.log(filing, filingType)
```

Comment:
I'm not sure if this console.log is required (since the rejection has a decent message already) though it may be useful to log the bad value.

If this console.log should remain then please add useful text in it so (a) user has a message of what went wrong, and (b) we can globally search the code for this message :)

Also, you should include `eslint-disable-line no-console` for log messages to avoid linter warnings/errors.

Reply:
Good thinking. I've updated the line with your suggestions.